lib/sen_tweet/bitfeels/metrics.ex

```diff
@@ -7,12 +7,16 @@ defmodule SenTweet.Bitfeels.Metrics do
   alias SenTweetWeb.MetricChannel
 
   def create_metrics do
-    %{
-      tweets_processed: 0,
-      sum_scores: 0,
-      average_score: 0,
-      histogram: Enum.map(0..10, &[-1 + 2 * &1 / 11, -1 + 2 * (&1 + 1) / 11, 0])
-    }
+    for type <- ["extended_tweet", "retweeted_status", "quoted_status", "text"],
```

Comment:
Excellent choice with for into here. I suggest pulling some of this out to get a nice oneliner.

`for type <- @tweet_types, into: %{}, do: {type, default_metrics()}`

Another option would be `Enum.into/3`

`Enum.into(@tweet_types, %{}, &{&1, default_metrics())`